src/features/world/scenes/RetreatScene.ts

```diff
@@ -119,12 +119,6 @@ export class RetreatScene extends BaseScene {
     });
     bigGoblin.play("big_goblin_animation", true);
 
-    const exchange = this.add.sprite(114, 215, "exchange");
-    // On click
-    exchange.setInteractive({ cursor: "pointer" }).on("pointerdown", () => {
-      interactableModalManager.open("goblin_market");
-    });
-
```

Comment:
We need all of this also. These interactables should just be on the building/npc so that we can interact with it.